modules/snooker/src/main/ts/ephox/snooker/util/CssUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { HTMLElement, HTMLTableElement } from '@ephox/dom-globals';
```

Comment:
Naming files is hard. Any other suggestions for this? Considered just merging it with the existing Utils file?

Reply:
Also this is very TinyMCE-specific in general - maybe shouldn't be moved?

Reply:
Have moved out the very TinyMCE-specific things and left the rest. Does this work?

Reply:
I can't see anything in this file that should really be in snooker tbh. I'd suggest leaving it in TinyMCE/Advanced Tables.

Reply:
Yeah, going to move it back